narwhal/Docker/Dockerfile

```diff
@@ -86,17 +86,17 @@ RUN cargo build --${BUILD_MODE} --features ${FEATURES} --bin node --bin benchmar
 FROM debian:bullseye-slim
 ARG BUILD_MODE
 
-WORKDIR "$WORKDIR/narwhal"
+WORKDIR "$WORKDIR/sui"
 
 # Copy the Narwhal node binary to bin folder
-COPY --from=builder narwhal/target/${BUILD_MODE}/node bin/
+COPY --from=builder sui/target/${BUILD_MODE}/node bin/
```

Comment:
We should rename the binary to something more descriptive from 'node'.  Maybe 'narwhal-node' ?

Reply:
Sounds good. Fixed all the affected locations that I can find.